.eslintrc

```diff
@@ -96,7 +96,7 @@ rules:
   radix: 2
   vars-on-top: 0
   wrap-iife: 2
-  yoda: 0
+  yoda: [1, never]
```

Comment:
This one was "error" in your Gist, is there cases where we can't adapt the code?